src/main/java/qupath/ext/align/gui/ImageAlignmentPane.java

```diff
@@ -1018,5 +1025,120 @@ protected void updateItem(ImageData<BufferedImage> item, boolean empty) {
 		
 		
 	}
+
+	private static class ProjectEntryListCell extends ListCell<ProjectImageEntry<BufferedImage>> {
+		
+		private Tooltip tooltip = new Tooltip();
+		private ImageView imageView = new ImageView();
+	
+		private ProjectEntryListCell() {
+			super();
+			imageView.setFitWidth(250);
+			imageView.setFitHeight(250);
+			imageView.setPreserveRatio(true);
+		}
 	
+		@Override
+		protected void updateItem(ProjectImageEntry<BufferedImage> item, boolean empty) {
+			super.updateItem(item, empty);
+			if (item == null || empty) {
+				setText(null);
+				setGraphic(null);
+				setTooltip(null);
+				return;
+			}
+			setText(item.getImageName());
+			
+			Node tooltipGraphic = null;
+			BufferedImage img = null;
+			try {
+				img = (BufferedImage)item.getThumbnail();
+				if (img != null) {
+					imageView.setImage(SwingFXUtils.toFXImage(img, null));
+					tooltipGraphic = imageView;
+				}
+			} catch (Exception e) {
+				logger.debug("Unable to read thumbnail for {} ({})" + item.getImageName(), e.getLocalizedMessage());
+			}
+			tooltip.setText(item.getSummary());
+			if (tooltipGraphic != null)
+				tooltip.setGraphic(tooltipGraphic);
+			else
+				tooltip.setGraphic(null);
+			setTooltip(tooltip);
+		}
+	}	
+
+	/**
+	 * We should just be able to call {@link ListSelectionView#getTargetItems()}, but in ControlsFX 11 there 
+	 * is a bug that prevents this being correctly bound.
+	 * @param <T>
+	 * @param listSelectionView
+	 * @return target items
+	 */
+	public static <T> ObservableList<T> getTargetItems(ListSelectionView<T> listSelectionView) {
+		var skin = listSelectionView.getSkin();
+		if (skin == null) {
+			return listSelectionView.getTargetItems();
+		}
+
+		try {
+			logger.debug("Attempting to access target list by reflection (required for controls-fx 11.0.0)");
+			var method = skin.getClass().getMethod("getTargetListView");
+			@SuppressWarnings("unchecked")
+			var view = (ListView<T>)method.invoke(skin);
+			return view.getItems();
+		} catch (Exception e) {
+			logger.warn("Unable to access target list by reflection, sorry", e);
+			return listSelectionView.getTargetItems();
+		}
+	}
+
+	/**
+	 * We should just be able to call {@link ListSelectionView#getSourceItems()}, but in ControlsFX 11 there 
```

Comment:
I think this workaround is no longer needed with ControlsFX 11.1.2 as used in QuPath now.